Makefile

```diff
@@ -0,0 +1,79 @@
+# Makefile to help automate tasks
+WD := $(shell pwd)
+PY := bin/python
+PIP := bin/pip
+PEP8 := bin/pep8
+PYTEST := bin/py.test
+
+
+# #######
+# INSTALL
+# #######
+.PHONY: all
+all: sysdeps venv
+
+.PHONY: clean-all
+clean-all: clean_venv
+	if [ -d dist ]; then \
+		rm -r dist; \
+    fi
+
+
+sysdeps:
+	sudo apt-get install python3-dev
+
+
+venv: bin/python
+bin/python:
+	# needs python3-dev to build keystoneclient deps
+	virtualenv -p /usr/bin/python3 .
```

Comment:
No, guess not. just that it's harder to source when you can't see the directory. the bin/etc is hidden. Updated.